src/components/20-molecules/top-content-bar/index.js

```diff
@@ -33,6 +36,9 @@ class AXATopContentBar extends LitElement {
   constructor() {
     super();
     applyDefaults(this);
+    /* eslint-disable no-undef */
+    defineVersioned([AXAButton, AXAButtonLink, AXAContainer], __VERSION_INFO__);
```

Comment:
@markus-walther i really really like how you solved this. It also has the implicit benefit of hard dependency locking

Reply:
Thanks!